optuna/trial.py

```diff
@@ -197,13 +204,28 @@ def suggest_loguniform(self, name, low, high):
             Suggest penalty parameter ``C`` of `SVC <https://scikit-learn.org/stable/modules/
             generated/sklearn.svm.SVC.html>`_.
 
-            .. code::
+            .. testcode::
+
+                import optuna
+                import numpy as np
+                from sklearn.svm import SVC
+                from sklearn.model_selection import train_test_split
 
-                >>> def objective(trial):
-                >>>     ...
-                >>>     c = trial.suggest_loguniform('c', 1e-5, 1e2)
-                >>>     clf = sklearn.svm.SVC(C=c)
-                >>>     ...
+                def objective(trial):
+                    c = trial.suggest_loguniform('c', 1e-5, 1e2)
+                    assert 1e-5 <= c < 1e2
+
+                    np.random.seed(seed=0)
+                    X = np.random.randn(50).reshape(-1, 1)
+                    y = np.random.randint(0, 2, 50)
+                    X_train, X_test, y_train, y_test = train_test_split(X, y, random_state=0)
```

Comment:
Sorry to change my mind, but this dataset preparation is commonly seen in other examples, so please hide it into the `.. testsetup::` block?

Current:
![image](https://user-images.githubusercontent.com/3255979/74005348-e05b1900-49bb-11ea-9fd1-d18228e03218.png)


Proposal:
![image](https://user-images.githubusercontent.com/3255979/74005267-a558e580-49bb-11ea-866a-1c33ed79fc2e.png)